tools/yaml2json.py

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import ruamel.yaml
+from pathlib import Path
 import json
 import glob
 import sys
-import os
 
 # grab the yaml files
 files = glob.glob('schemas/input/csl-*.yaml')
```

Comment:
pathlib.Path objects have a .glob method if you want to get rid of another import.